### ISSUE TITLE
Real life accuracy & bug in parse_match_data

### ISSUE THREAD

**Issue by m-kloeckner** [brakid/BundesligaPrediction#2]
This is a very interesting project! Thank you very much for sharing this. Can you provide some experience on the real life accuracy of your model? Did you work on this any further?

Why am I asking: The 99% accuracy in training and validation actually feels a bit odd. When reviewing the code I found a bug in line 114 of prediction.py. It seems that the match data is constructed of multiple copies of year 2010's match data. This would result in identical data for training an evaluation - which would in turn explain the high accuracy.  

See https://github.com/brakid/BundesligaPrediction/blob/2b647f2b8a585b8dcbcfb91ff21434cf48b68ee0/prediction.py#L114

**Comment by brakid** [comment]
Good catch, indeed. This is a mistake that indeed can be responsible for the super-high accuracy in training and validation.

I did not test the model further (unfortunately), but would be happy to hear more if you are trying it out in more detail.
